libdnf/sack/query.cpp

```diff
@@ -63,20 +63,42 @@ struct default_delete<DnfPackage> {
 namespace libdnf {
 
 struct NevraID {
-    Id name;
-    Id arch;
-    Id evr;
+public:
+    NevraID();
+    NevraID(const NevraID & src);
+    NevraID(NevraID && src);
+    NevraID & operator=(const NevraID & src);
+    NevraID & operator=(NevraID && src);
+    Id name{0};
+    Id arch{0};
+    Id evr{0};
+    std::string evr_str;
```

Comment:
Personally I don't like that it initialize empty string and then the object is replaced by another object. The question if I should replace it by char *? It will spare initialization and call evr_str.c_str() when it compares version. @jrohel what do you think?